_data/next.yml

```diff
@@ -16,38 +16,37 @@
 
 cards:
   done:
-    - title: AMA on r/privacytoolsio
+    - title: Safing API Backend
       description:
-        - Reach out to the PTIO team. Together, schedule and host an IAMA event so people can ask us anything.
-      projects:
-        - Community
-      team:
-        - David
-        - Raphael
-    - title: Portscan Detection
-      description: Detect and block malicious actors trying to expose vulnerabilities by scanning through all networking ports on your device. Research thoroughly to prevent false positives.
+        - Enable you to sign in to your Safing account on the Portmaster client via the Safing API. Also respond with details regarding your SPN subscription.
+        - Implement device management so you can sign out of devices from the Safing website.
       projects:
+        - SPN
         - Portmaster
+        - Website
       team:
-        - Daniel
-    - title: Administration Interface
+        - David
+    - title: "Inception: Reaching Out"
       description:
-        - Enable admins to easily accept cash payments, manage refunds or suspend rogue users.
-        - Also provide some basic insights on signups, payments, survey results, etc...
+        - As the SPN will saw the light of day, the time has come to spread the word, also by spending money.
```

Comment:
also by spending money. -> via word of mouth and buying advertisements.

das sollte expliziter sein, wir bestechen niemanden.

Reply:
what about: "via word of mouth and paid marketing"?

Reply:
ja passt